recipes/open62541/all/conanfile.py

```diff
@@ -31,7 +31,8 @@ class Open62541Conan(ConanFile):
         "shared": [True, False],
         "historize": [True, False, "Experimental"],
         "logging_level": ["Fatal", "Error", "Warning", "Info", "Debug", "Trace"],
-        "subscription": [True, False, "With Events"],
+        # "With Events" is deprecated
+        "subscription": [True, False, "With Events", "events", "events+alarms+conditions"],
```

Comment:
When using commas (but without spaces in between), this makes this option the same interace as the `package` option of msys2.
```suggestion
        "subscription": [True, False, "With Events", "events", "alarms,conditions,events"],
```